package.json

```diff
@@ -16,8 +16,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@brandonocasey/spawn-promise": "~0.2.0",
-    "colorette": "^1.2.2",
-    "es-check": "^5.2.3",
+    "colorette": "^2.0.16",
+    "es-check": "^6.1.1",
     "exit-hook": "^2.2.1",
```

Comment:
this package cannot be updated without rewriting this package to use native es-modules.